sdk/schemaregistry/azure-schemaregistry/CHANGELOG.md

```diff
@@ -12,7 +12,11 @@
   - `group_name`, which has been renamed from `schema_group`
   - `name`, which has been renamed from `schema_name`
   - `content`, which has been renamed from `schema_content`
-  - `serialization_type`
+  - `format`, which has been renamed from `serialization_type`
+- `location` instance variable in `SchemaProperties` has been removed.
+- The following positional arguments are now required keyword arguments:
+  - `id`, `serialization_type`, and `version` on `SchemaProperties`
+  - `content` and `properties` on `Schema`
```

Comment:
oh.. I think we just don't want to let users construct them at all -- just showing the ivars docstring, no keyword docstring.

```python
    :ivar id: References specific schema in registry namespace.
    :type id: str
    :ivar location: URL location of schema, identified by schema group, schema name, and version.
    :ivar serialization_type: Serialization type for the schema being stored.
    :type serialization_type: str
    :ivar version: Version of the returned schema.
    :type version: int
class SchemaProperties(object):
    def __init__(self, **kwargs):
        ...
```